### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by frinyvonnick**
### Prerequisites
- [x] I have read the [Contributing guidelines](https://github.com/Zenika/immutadot/blob/master/.github/CONTRIBUTING.md)
- [x] I have read the [Code of conduct](https://github.com/Zenika/immutadot/blob/master/.github/CODE_OF_CONDUCT.md) and I agree with it

### Description

- [x] Add a section about `interoperability`in `our approach`
- [x] Installation
- [x] Usage
- [x] Documentation 
- [x] Contributing 
- [x] License

**Issue :** #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ An immutable object is an object that cannot be changed once created. It brings
 
 [ES2015+](https://github.com/tc39/ecma262#ecmascript) new features are great to deal with arrays and objects. As data structures get deeper, the code you write to make data immutable gets bigger and less readable. immutad●t uses the dot notation to address this issue.
 
+### Interoperability
+
+Since immutad●t uses plain JavaScript objects you can use all standards ways to access your data. Moreover it lets you use all your favorites libraries.
```
Comment (charlyx) [inline README.md]: immutad●t uses plain JavaScript objects so you can access your data using standard ways. Moreover, it lets you freely enjoy your favourites libraries.

File: README.md [hunk 2]
```diff
@@ -42,6 +46,66 @@ immutad●t comes with a large set of built-in utilities, mostly based on [lodas
...
+
+  push({ nested: { prop: [1, 2] } }, 'nested.prop', 3, 4)
+  // → { nested: { prop: [1, 2, 3, 4] } }
+```
+
+Feel free to try immutad●t with our online [demo](https://npm.runkit.com/immutadot).
+
+## Documentation
+
+You can find out a full list of immutad●t's features in our [documentation](https://zenika.github.io/immutadot).
```
Comment (charlyx) [inline README.md]: There is a full list of immutad●t's features

File: README.md [hunk 3]
```diff
@@ -42,6 +46,66 @@ immutad●t comes with a large set of built-in utilities, mostly based on [lodas
...
+
+## Contributing
+
+We want contributing to immutad●t to be fun, enjoyable, and educational for anyone, and everyone.
+
+### [Code of Conduct](https://github.com/Zenika/immutadot/blob/master/.github/CODE_OF_CONDUCT.md)
+
+In the interest of fostering an open and welcoming environment, we have adopted a Code of Conduct that we expect project participants to adhere to. Please read the [full text](https://github.com/Zenika/immutadot/blob/master/.github/CODE_OF_CONDUCT.md) so that you can understand what actions will and will not be tolerated.
```
Comment (charlyx) [inline README.md]: In the interest of fostering an open and welcoming environment, we have adopted a Code of Conduct that we expect people to commit to.

Comment (charlyx) [inline README.md]: Please read the full text so you can understand what behaviour will and will not be tolerated.

File: README.md [hunk 4]
```diff
@@ -34,6 +34,10 @@ An immutable object is an object that cannot be changed once created. It brings
 
 [ES2015+](https://github.com/tc39/ecma262#ecmascript) new features are great to deal with arrays and objects. As data structures get deeper, the code you write to make data immutable gets bigger and less readable. immutad●t uses the dot notation to address this issue.
```
Comment (sonyth) [inline README.md]: suggestion: as data structures expand...

File: README.md [hunk 5]
```diff
@@ -42,6 +46,66 @@ immutad●t comes with a large set of built-in utilities, mostly based on [lodas
...
+
+We want contributing to immutad●t to be fun, enjoyable, and educational for anyone, and everyone.
+
+### [Code of Conduct](https://github.com/Zenika/immutadot/blob/master/.github/CODE_OF_CONDUCT.md)
+
+In the interest of fostering an open and welcoming environment, we have adopted a Code of Conduct that we expect project participants to adhere to. Please read the [full text](https://github.com/Zenika/immutadot/blob/master/.github/CODE_OF_CONDUCT.md) so that you can understand what actions will and will not be tolerated.
+
+### [Contributing guide](https://github.com/Zenika/immutadot/blob/master/.github/CONTRIBUTING.md)
+
+If you are interested in contributing to immutad●t, please read our [contributing guide](https://github.com/Zenika/immutadot/blob/master/.github/CONTRIBUTING.md) to learn more about how to propose bugfixes and improvements.
```
Comment (sonyth) [inline README.md]: to learn more about how to suggest bugfixes...

File: README.md (hunk 1 above)
Comment (sonyth) [inline README.md]: favorite libraries

File: README.md (hunk 2 above)
Comment (nlepage) [inline README.md]: What about : "The full list of immutad●t's features is available in our documentation."

File: README.md [hunk 6]
```diff
@@ -42,6 +46,66 @@ immutad●t comes with a large set of built-in utilities, mostly based on [lodas
 
 If you are already familiar with [lodash](https://lodash.com/) and [ES2015+](https://github.com/tc39/ecma262#ecmascript) then you should be able to use immutad●t quickly.
 
+## Installation
+
+immutad●t is available on [npm repository](https://www.npmjs.com/package/immutadot).
+
+using npm:
```
Comment (nlepage) [inline README.md]: yarn first please

File: README.md [hunk 7]
```diff
@@ -42,6 +46,66 @@ immutad●t comes with a large set of built-in utilities, mostly based on [lodas
...
+In the interest of fostering an open and welcoming environment, we have adopted a Code of Conduct that we expect project participants to adhere to. Please read the [full text](https://github.com/Zenika/immutadot/blob/master/.github/CODE_OF_CONDUCT.md) so that you can understand what actions will and will not be tolerated.
+
+### [Contributing guide](https://github.com/Zenika/immutadot/blob/master/.github/CONTRIBUTING.md)
+
+If you are interested in contributing to immutad●t, please read our [contributing guide](https://github.com/Zenika/immutadot/blob/master/.github/CONTRIBUTING.md) to learn more about how to propose bugfixes and improvements.
+
+## License
+
+immutad●t is [MIT licensed](https://github.com/Zenika/immutadot/blob/master/LICENSE.md).
+
```
Comment (nlepage) [inline README.md]: Add a Credits paragraph

**Pull request merged**